src/pages/admin/TicketCards.vue

```diff
@@ -0,0 +1,450 @@
+          :text-area="true"
+          :value="TicketReply"
+          class="w-11/12 m-4"
+          @input="
+            (value) => {
+              TicketReply = value;
+            }
+          "
+        ></base-input>
+
+        <hr />
+        <div class="flex justify-between">
+          <div class="flex flex-col">
+            <div class="flex justify-between p-2">
+              <img
+                class="mr-4"
+                style="width: 50px; height: 50px;"
+                :src="assignWhoPicture"
+              />
+              <div class="flex flex-col mr-4">
+                Assigned To:
+                <div v-if="ticketData.assignee_id">
+                  {{ assignWho }}
+                </div>
+                <div v-else>No user Assigned</div>
+              </div>
+
+              <form-select
+                class="w-auto h-12 flex-grow border border-crisiscleanup-dark-100 select"
+                :options="userList"
+                :value="selectedUser"
+                @input="
+                  (value) => {
+                    selectedUser = value;
+                  }
+                "
+              />
+              <base-button
+                text="Reassign Ticket"
+                variant="solid"
+                class="m-2 p-1"
+                :action="assignUser"
+              />
+            </div>
+            <div class="flex justify-evenly">
+              <base-button
+                style="background: #9cb8ff;"
+                text="Reply as Open"
+                variant="solid"
+                class="m-2 p-1"
+                :action="replyToTicketOpen"
+              />
+
+              <base-button
+                style="background: #e8e4e4;"
+                text="Reply as Pending"
+                variant="solid"
+                class="m-2 p-1"
+                :action="replyToTicketPending"
+              />
+
+              <base-button
+                style="background: #ffa296;"
+                text="Reply as Solved/Closed"
+                variant="solid"
+                class="m-2 p-1"
+                :action="replyToTicketSolved"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import moment from 'moment';
+
+import User from '@/models/User';
+import FormSelect from '../../components/FormSelect.vue';
+
+export default {
+  name: 'TicketCards',
+  filters: {
+    formatDate(value) {
+      if (value) {
+        return moment(String(value)).format('MM/DD/YYYY hh:mm');
+      }
+      return null;
+    },
+    truncate(data, num) {
+      const reqdString = data.split('').slice(0, num).join('');
+      return reqdString;
+    },
+  },
+  props: {
+    ticketData: {
+      type: Object,
+      default: null,
+    },
+    expandState: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  components: { FormSelect },
+  async mounted() {
+    await this.currentUser();
```

Comment:
Not sure why this is an action instead of computed.

But regardless, you could just do it like this:
https://github.com/CrisisCleanup/crisiscleanup-3-web/blob/df14f6dd1bddfdc14e7e0eaa7ee4afa0d2285768/src/mixins/user.js#L36-L39

(or, just use that mixin I just linked to and have it do it for you :shrug: )